ts/features/idpay/initiative/details/components/BeneficiaryDetailsContent.tsx

```diff
@@ -135,10 +137,13 @@ const BeneficiaryDetailsContent = (props: BeneficiaryDetailsProps) => {
       )
     );
 
+  const initiativeType = useIOSelector(idPayInitiativeTypeSelector);
+
```

Comment:
What happens if there is no initiative in the store? Wouldn't be better pass the type with the rest of data via route parameters?